src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/LongPollingTransport.cs

```diff
@@ -73,7 +71,7 @@ private async Task ProcessAsync(Uri url)
         {
             // Start sending and polling (ask for binary if the server supports it)
             var receiving = Poll(url, _transportCts.Token);
-            var sending = SendUtils.SendMessages(url, _application, _httpClient, _logger);
+            var sending = SendUtils.SendMessages(url, _application!, _httpClient, _logger);
```

Comment:
To make `_application` not null wherever you use it in this method, you can add `Debug.Assert(_application != null)` at the start of the method.